src/contrib/libbpf/include/uapi/linux/if_xdp.h

```diff
@@ -85,7 +85,7 @@ struct xdp_options {
 /* Flags for the flags field of struct xdp_options */
 #define XDP_OPTIONS_ZEROCOPY (1 << 0)
 
-/* Pgoff for mmaping the rings */
+/* Pgoff for mmapping the rings */
```

Comment:
the function is `mmap()`, based on `memory-map` the conjugation of `map` with `-ing` is `mapping`.